### ISSUE TITLE
time out request

### ISSUE THREAD

**Issue by meeki007** [dkern/node-red-throttle#8]
exampe:
set by count.
count limit 2
if count ≠ 2 within 3min clear count

**Comment by eisbehr-** [comment]
Hello @meeki007 

Can you give me some more details of what you want to tell me? Is this an issue, an feature you want or what else?!

Thanks!

**Comment by meeki007** [comment]
Hello :)

I made a node to do what I needed.

https://github.com/meeki007/node-red-contrib-timeframerlt/blob/master/readme.md

I used a bunch of your code. Threw a credit link to your node in my js file
and at the bottom of GitHub readme.md web page. Also published it to npm so
everyone can enjoy.

Company wanted the functionality and okayed a novice bash,  C# guy (me) to
dig into js. Just had to learn the syntax and learn how to get around js
lacking a Logical XOR.

Adding a blue square at the bottom of the node I made for count status on
my free time.

I kinda know your code inside and out now. Want me to see if I can add blue
status square to your node like a previous user requested for yours? Pull
request?

Oh and I'm using your node for a flow as well so thanks for your work.

Thanks,

David L Burrows

**Comment by eisbehr-** [comment]
Hello David,

thanks for your answer. Glad you archived what you wanted. But why not integrated your needs into the exiting throttle node? ;) Now there are two throttle nodes. I think it would be pretty easy and nice to integrate in the existing node.

**Comment by eisbehr-** [comment]
Yes, this would be great, if you would like to implement this. We could create a second branch for your changes, so we could work on it together, till it's ready for release!
